packages/web_benchmarks/lib/client.dart

```diff
@@ -134,12 +132,12 @@ void _fallbackToManual(String error) {
 
 /// Visualizes results on the Web page for manual inspection.
 void _printResultsToScreen(Profile profile) {
-  html.document.body.innerHtml = '<h2>${profile.name}</h2>';
+  html.document.body!.innerHtml = '<h2>${profile.name}</h2>';
```

Comment:
If you're going to use something that needs to be force-unwrapped multiple times in the same method, please make a local variable for it that's non-nullable.